.github/workflows/scripts/countline.py

```
@@ -0,0 +1,295 @@
+    result = True
+
+    # Test
+    for invalid_extension in invalid_extensions:
+        if filepath.lower().endswith(invalid_extension.lower()) is False:
+            continue
+        result = False
+
+    return result
+
+
+def _valid_exclusions(excludes):
```

Ruff (0.12.2)

Missing return type annotation for private function `_valid_exclusions`

(ANN202)